README.md

```diff
@@ -145,6 +145,8 @@ module "serverless" {
   service_name = "sparklepants"
   stage        = "${var.stage}"
 
+  # lambda_role_name    = "your-custom-role"
```

Comment:
NIT: This should go below `# (Default values)` in the right order with default value (and maybe suggestion after that).